### PR TITLE
Readme Updated

### PR DESCRIPTION
**Pull request by sampoearas**
- Updated Readme format to table style. 
- Fixed broken image links. Added catch for malformatted pics.

### CODE REVIEW

File: Innovator/Client/scripts/kanban/js/script.js [hunk 1]
```diff
@@ -453,16 +459,17 @@ function loadSingleBoard(boardId, reload) {
 
 		// get cardimage
 		var vaultref = item.getProperty("managerpicture");
-		var fileUrl = "nomanager.png";
+		var fileUrl = "../kanban/nomanager.png";
 		if ((vaultref !== undefined) && (vaultRefTable[vaultref] == undefined)) {
-			if (inn.getFileUrl(vaultref.split("?fileId=")[1], 1) == undefined) {
+			try{fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
```
Comment (EliJDonahue) [inline Innovator/Client/scripts/kanban/js/script.js]: Fix the formatting of the try-catch statement.

File: Innovator/Client/scripts/kanban/js/script.js [hunk 2]
```diff
@@ -235,11 +235,17 @@ function loadBoards() {
...
 		// get boardimage
 		var vaultref = item.getProperty("creatorpicture","undefined");
-		var fileUrl = "nomanager.png";
-		/*if ((vaultref !== "undefined") && (vaultRefTable[vaultref] == undefined)) {
-			fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
-			vaultRefTable[vaultref] = fileUrl;
-		}*/
+		var fileUrl = "../kanban/nomanager.png";
+		if ((vaultref !== "undefined") && (vaultRefTable[vaultref] == undefined)) {
+			try{fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
```
Comment (EliJDonahue) [inline Innovator/Client/scripts/kanban/js/script.js]: Fix the formatting of the try-catch statement.

File: Innovator/Client/scripts/kanban/js/script.js [hunk 3]
```diff
@@ -235,11 +235,17 @@ function loadBoards() {
...
+				vaultRefTable[vaultref] = fileUrl;
+				}
+			catch(err){
+				console.log("Profile Image for " + item.getPropertyAttribute("created_by_id", "keyed_name") + " is malformed. Please choose a different image.");
+				}
+			
+		}
+		else{vaultRefTable[vaultref] = fileUrl;}
```
Comment (EliJDonahue) [inline Innovator/Client/scripts/kanban/js/script.js]: The format of our JavaScript code blocks should always look like this:
```
if (condition) {
    // code
} else {
    // code
}
```
Code blocks should not have closing brackets at the end of a line - they should always go on a new line after the block they're closing. This applies to if-else statements, try-catch blocks, and functions.

**Review by EliJDonahue — CHANGES_REQUESTED** [review]

File: Innovator/Client/scripts/kanban/js/script.js (hunk 1 above)
Comment (sampoearas) [reply Innovator/Client/scripts/kanban/js/script.js]: fixed.

File: Innovator/Client/scripts/kanban/js/script.js (hunk 2 above)
Comment (sampoearas) [reply Innovator/Client/scripts/kanban/js/script.js]: fixed.

File: Innovator/Client/scripts/kanban/js/script.js (hunk 3 above)
Comment (sampoearas) [reply Innovator/Client/scripts/kanban/js/script.js]: fixed.

**Comment by sampoearas** [comment]
Issues Resolved.

File: Innovator/Client/scripts/kanban/js/script.js [hunk 4]
```diff
@@ -453,14 +462,18 @@ function loadSingleBoard(boardId, reload) {
...
-				vaultRefTable[vaultref] = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
-			}
-		} else if (vaultref == undefined) {
+				}
+			catch(err){
+				console.log("Profile Image for " + item.getPropertyAttribute("manager", "keyed_name") + " is malformed. Please choose a different image.");
+				}
+			
+		}
+		else {
```
Comment (EliJDonahue) [inline Innovator/Client/scripts/kanban/js/script.js]:
```suggestion
		} else {
```

File: Innovator/Client/scripts/kanban/js/script.js [hunk 5]
```diff
@@ -453,14 +462,18 @@ function loadSingleBoard(boardId, reload) {
...
-			} else {
-				vaultRefTable[vaultref] = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
-			}
-		} else if (vaultref == undefined) {
+				}
+			catch(err){
+				console.log("Profile Image for " + item.getPropertyAttribute("manager", "keyed_name") + " is malformed. Please choose a different image.");
+				}
+			
+		}
```
Comment (EliJDonahue) [inline Innovator/Client/scripts/kanban/js/script.js]:
```suggestion
```

File: Innovator/Client/scripts/kanban/js/script.js [hunk 6]
```diff
@@ -453,14 +462,18 @@ function loadSingleBoard(boardId, reload) {
...
-			if (inn.getFileUrl(vaultref.split("?fileId=")[1], 1) == undefined) {
+			try{
+				fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
 				vaultRefTable[vaultref] = fileUrl;
-			} else {
-				vaultRefTable[vaultref] = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
-			}
-		} else if (vaultref == undefined) {
+				}
+			catch(err){
```
Comment (EliJDonahue) [inline Innovator/Client/scripts/kanban/js/script.js]:
```suggestion
			} catch (err) {
```

File: Innovator/Client/scripts/kanban/js/script.js [hunk 7]
```diff
@@ -453,14 +462,18 @@ function loadSingleBoard(boardId, reload) {
...
 		if ((vaultref !== undefined) && (vaultRefTable[vaultref] == undefined)) {
-			if (inn.getFileUrl(vaultref.split("?fileId=")[1], 1) == undefined) {
+			try{
+				fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
 				vaultRefTable[vaultref] = fileUrl;
-			} else {
-				vaultRefTable[vaultref] = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
-			}
-		} else if (vaultref == undefined) {
+				}
```
Comment (EliJDonahue) [inline Innovator/Client/scripts/kanban/js/script.js]:
```suggestion
```

File: Innovator/Client/scripts/kanban/js/script.js [hunk 8]
```diff
@@ -453,14 +462,18 @@ function loadSingleBoard(boardId, reload) {
 
 		// get cardimage
 		var vaultref = item.getProperty("managerpicture");
-		var fileUrl = "nomanager.png";
+		var fileUrl = "../kanban/nomanager.png";
 		if ((vaultref !== undefined) && (vaultRefTable[vaultref] == undefined)) {
-			if (inn.getFileUrl(vaultref.split("?fileId=")[1], 1) == undefined) {
+			try{
```
Comment (EliJDonahue) [inline Innovator/Client/scripts/kanban/js/script.js]:
```suggestion
			try {
```

File: Innovator/Client/scripts/kanban/js/script.js [hunk 9]
```diff
@@ -235,11 +235,20 @@ function loadBoards() {
...
+			try {
+				fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
+				vaultRefTable[vaultref] = fileUrl;
+				}
+			catch(err){
+				console.log("Profile Image for " + item.getPropertyAttribute("created_by_id", "keyed_name") + " is malformed. Please choose a different image.");
+				}
+			
+		}
+		else {
```
Comment (EliJDonahue) [inline Innovator/Client/scripts/kanban/js/script.js]:
```suggestion
		} else {
```

File: Innovator/Client/scripts/kanban/js/script.js [hunk 10]
```diff
@@ -235,11 +235,20 @@ function loadBoards() {
...
+		if ((vaultref !== "undefined") && (vaultRefTable[vaultref] == undefined)) {
+			try {
+				fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
+				vaultRefTable[vaultref] = fileUrl;
+				}
+			catch(err){
+				console.log("Profile Image for " + item.getPropertyAttribute("created_by_id", "keyed_name") + " is malformed. Please choose a different image.");
+				}
+			
+		}
```
Comment (EliJDonahue) [inline Innovator/Client/scripts/kanban/js/script.js]:
```suggestion
```

File: Innovator/Client/scripts/kanban/js/script.js [hunk 11]
```diff
@@ -235,11 +235,20 @@ function loadBoards() {
...
-		var fileUrl = "nomanager.png";
-		/*if ((vaultref !== "undefined") && (vaultRefTable[vaultref] == undefined)) {
-			fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
+		var fileUrl = "../kanban/nomanager.png";
+		if ((vaultref !== "undefined") && (vaultRefTable[vaultref] == undefined)) {
+			try {
+				fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
+				vaultRefTable[vaultref] = fileUrl;
+				}
+			catch(err){
```
Comment (EliJDonahue) [inline Innovator/Client/scripts/kanban/js/script.js]:
```suggestion
			} catch(err) {
```

File: Innovator/Client/scripts/kanban/js/script.js [hunk 12]
```diff
@@ -235,11 +235,20 @@ function loadBoards() {
...
 		var vaultref = item.getProperty("creatorpicture","undefined");
-		var fileUrl = "nomanager.png";
-		/*if ((vaultref !== "undefined") && (vaultRefTable[vaultref] == undefined)) {
-			fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
+		var fileUrl = "../kanban/nomanager.png";
+		if ((vaultref !== "undefined") && (vaultRefTable[vaultref] == undefined)) {
+			try {
+				fileUrl = inn.getFileUrl(vaultref.split("?fileId=")[1], 1);
+				vaultRefTable[vaultref] = fileUrl;
+				}
```
Comment (EliJDonahue) [inline Innovator/Client/scripts/kanban/js/script.js]:
```suggestion
```

**Review by EliJDonahue — CHANGES_REQUESTED** [review]
The closing curly bracket before "catch" and "else" should be on the same line.
```
if (condition) {
    // ...
} else {
    // ...
}
```

**Comment by sampoearas** [comment]
Suggestions Applied.

**Review by EliJDonahue — APPROVED** [review]

**Pull request merged**